vista_2d/Quickstart_guide.md

```diff
@@ -0,0 +1,100 @@
+pip install pynvml==11.5.0 #optional for MLFlow support
+```
+- If use local environment, make sure to install following libraries.
+```
+pip install torch==2.2.2 monai==1.3.1 numpy==1.24.4
+pip install fire==0.6.0 tifffile==2024.5.10 imagecodecs==2024.1.1 pillow==10.2.0 fastremap==1.14.1 numba==0.59.0
+pip install --no-deps cellpose==3.0.8 natsort==8.4.0 roifile==2024.5.24
+pip install git+https://github.com/facebookresearch/segment-anything.git@6fdee8f2727f4506cfbbe553e23b895e27956588#egg=segment_anything
+pip install mlflow==2.13.1 psutil==5.9.4 pynvml==11.5.0 #optional for MLFlow support
+```
+
+#### 2.) Obtain a public dataset Or Create a custom dataset JSON
+
+###### Retrieve Cellpose public dataset
+
+Trying on your or a new dataset is encouraged. If you do not have access to data, the [Cellpose dataset](https://www.cellpose.org/dataset)
+is fairly small and quick to download. Below are instructions on how to download the data
+
+![cellpose_links.png](cellpose_links.png)
+Click on train.zip and test.zip to download both directories independently. They both need to be
+placed in a `cellpose_dataset` directory. The `cellpose_dataset` will have to be created by the user
+in the root data directory. For e.g we can use `/user/cellpose_dataset`.
+
+Ensure that both `train.zip` & `test.zip` have been extracted from the zip format. Below is an example screenshot
+of how cellpose dataset directory should look like.
+
+![cellpose_dir.png](cellpose_dir.png)
+
+##### Creating JSON for custom datasets
+- The Vista-2D bundle uses JSON file lists for training and running inference on models. All JSON datalists that were
+used for Vista-2D training can be found in `datalists` directory
+- JSON format has mandatory requirements for a `training` list. The list has a dictionary
+per data sample that contains 3 keys `image`, `label`, `fold`. `image` contains the relative filename/filepath of the input image, the `label`
+contains the instance segmentation filename and `fold` represents if the image will be used for validation or not depending upon which fold is chosen.
+- The JSON's use an indent of 4 spaces
+- Refer `scripts/generate_json_cellpose.py` for an example on how to create a JSON file
+- Please note that once a JSON file is prepped it can be placed in `datalists` directory. The `config/hyper_parameters.yaml` file
+will need to be updated with the data root path as the `basedir` and the path to new JSON file is required at `datalists` variable in yaml file.
+
+#### 3.) Set Paths for data & execute inference
+- Set the `basedir` variable to the dataset path in the `cell_vista_segmentation/configs/quick_hyper_params.yaml` file.
+- If infer with your own model, use following command.
+ ```bash
+  python -m monai.bundle run_workflow "scripts.workflow.VistaCell" --config_file configs/quick_hyper_params.yaml --mode infer --pretrained_ckpt_name model.pt
```

Comment:
Hi @finalelement ,

I think we should develop a quick notebook to show how to use the new VISTA2D APIs instead of a usage guide of VISTA2D bundle, it should be in the README of VISTA2D bundle.

Thanks.